content/manuals/offload/configuration.md

```diff
@@ -0,0 +1,109 @@
+---
+title: Configure Docker Offload
+linktitle: Configure
+weight: 20
+description: Learn how to configure build settings for Docker Offload.
+keywords: cloud, configuration, settings, cloud builder, GPU, disk allocation, private resources, firewall
+---
+
+To use Docker Offload, you must start it in Docker Desktop. For more details,
+see the [Docker Offload quickstart](/offload/quickstart/).
+
+Settings for the cloud builders in Docker Offload can be further configured, in
+addition to settings for an entire organization, through **Cloud settings** in
+the Docker Cloud dashboard.
+
+> [!NOTE]
+>
+> To view usage and configure billing for Docker Offload, see [Docker Offload
+> usage and billing](/offload/usage/).
+
+## Cloud settings
+
+The **Cloud settings** page in Docker Home lets you configure disk
+allocation, private resource access, and firewall settings for your cloud
+builders in your organization.
+
+To view the **Cloud settings** page:
+
+1. Go to [Docker Home](https://app.docker.com/).
+2. Select the account for which you want to manage Docker Offload.
+3. Select **Cloud** > **Cloud settings**.
+
+The following sections describe the available settings.
+
+### Disk allocation
+
+The **Disk allocation** setting lets you control how much of the available
+storage is dedicated to the build cache. A lower allocation increases storage
+available for active builds.
+
+To make disk allocation changes, navigate to **Cloud settings** in Docker
+Cloud and then adjust the **Disk allocation** slider to specify the
+percentage of storage used for build caching.
+
+Any changes take effect immediately.
+
+> [!TIP]
+>
+> If you build very large images, consider allocating less storage for caching.
+
+### Build cache space
+
+Your subscription includes the following Build cache space:
+
+| Subscription | Build cache space |
+|--------------|-------------------|
+| Personal     | N/A               |
+| Pro          | 50GB              |
+| Team         | 100GB             |
+| Business     | 200GB             |
+
+To get more Build cache space, [upgrade your subscription](/manuals/subscription/change.md).
+
+> [!TIP]
```

Comment:
The tip sounds repetitive as it's also documented right above the Build cache space section. Suggest we remove one of these.